src/amo/middleware/prefixMiddleware.js

```diff
@@ -99,9 +99,7 @@ export function prefixMiddleware(req, res, next, { _config = config } = {}) {
   if (newURL !== req.originalUrl && !newURL.startsWith('//')) {
     // Collect vary headers to apply to the redirect
     // so we can make it cacheable.
-    if (isLangFromHeader) {
-      res.vary('accept-language');
-    }
+    res.vary('accept-language');
```

Comment:
This would add the header for pages that redirect for non-language reasons (e.g. /en-GB/ to /en-GB/firefox/) too, which is okay?

Reply:
Yeah, I don't think it's worth the hassle.